mne/io/proj.py

```diff
@@ -100,6 +100,13 @@ def add_proj(self, projs, remove_existing=False):
 
         return self
 
+    def add_eeg_ref(self):
+        """Add an average EEG reference projector if one does not exist
+        """
+        if _needs_eeg_average_ref_proj(self.info):
+            eeg_proj = make_eeg_average_ref_proj(self.info, activate=True)
```

Comment:
yeah, that's what I thought. Although, `activate` is slightly confusing to me. I would say `mark_active` or something like that. Because `activate` seems to give the impression that something was done to the data ... but that's a separate discussion altogether.